HomeWork_One/Quiz_Template/src/components/buttons/Button.tsx

```diff
@@ -0,0 +1,13 @@
+import './Button.css'
+import {Link} from "react-router-dom";
+export function Button({title,route,click}) {
+    return(
+        <>
+            <Link onClick={click} to={route}>
+                <button className='btn'>{title}</button>
+            </Link>
+
+        </>
+    )
+}
```

Comment:
Why don't we use `onClick` for the button element? I think it would be more generic since you've called this component a Button
e.g. by clicking the Button I need to send a request to the server or calculate something and update the store
I think in this case you'll need to refactor this component, right? WDYT?